app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -132,6 +134,10 @@ class BrowserFragment : IWebViewLifecycleFragment(), BrowserNavigationOverlay.Na
         cursor = null
     }
 
+    override fun isBackEnabled() = canGoBack()
```

Comment:
The reason I have a remove todo on these is because they should be called directly on the webView, not the browserFragment. This:
1) Limits BrowserFragment's reponsibility
2) Better encapsulates WebView functionality
3) Prevents BrowserFragment from getting bloated from all of its contained functionality (Home screen, WebView, Cursor, full screen videos, etc.)

So I'd prefer this to be set on the WebView instead.

That being said, there are some reasons for having some functionality in BrowserFragment:
1) The functionality exceeds WebViews responsibilities, e.g. pressing back may go back in the WebView, or it might pop the fragment stack to show home
2) It allows easier access through property observers (e.g. we want to update multiple components when a value changes)

Reply:
Okay, @mcomella but in this case, if we can just make `canGoBack` and `goBack` private, then we can use them right? I agree with you that they don't need to be exposed but I've been using them as convenience functions, and I think that comment is confusing.